pkgs/development/python-modules/qpsolvers/default.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildPythonPackage,
+  flit,
+  unittestCheckHook,
+  daqp,
+  ecos,
+  numpy,
+  osqp,
+  scipy,
+  scs,
+  quadprog,
+}:
+buildPythonPackage rec {
+  pname = "qpsolvers";
+  version = "3.4.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "qpsolvers";
+    repo = "qpsolvers";
+    rev = "v${version}";
+    hash = "sha256-GrYAhTWABBvU6rGoHi00jBa7ryjCNgzO/hQBTdSW9cg=";
+  };
+
+  nativeBuildInputs = [flit unittestCheckHook];
+
+  propagatedBuildInputs = [
+    daqp
+    ecos
+    numpy
+    osqp
+    scipy
+    scs
+
+    # For unittest
+    quadprog
+  ];
+
+  buildPhase = ''
+    flit build
+  '';
+
+  meta = with lib; {
+    description = "Quadratic programming solvers in Python with a unified API";
+    homepage = "https://github.com/qpsolvers/qpsolvers";
+    license = licenses.lgpl3;
```

```suggestion
    license = licenses.lgpl3Plus;
```
https://github.com/qpsolvers/qpsolvers/blob/00545e02615a5efbfea4ba4dd3a324e2da77e4e0/qpsolvers/__init__.py#L10-L11